src/main/java/thePackmaster/cards/monsterhunterpack/CarvingKnife.java

```diff
@@ -56,7 +56,6 @@ public void update() {
     public CarvingKnife() {
         super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
         baseDamage = damage = DAMAGE;
-        FleetingField.fleeting.set(this, true);
         tags.add(CardTags.HEALING);
```

Comment:
No exhaust?
I think the exhaust might still be nice just so you can't spam this card all the time